ci/build.sh

```diff
@@ -72,8 +72,8 @@ function set_up_github() {
   local token="${1}"
   local branch="${2}"
 
-  git config --global user.email "travis@travis-ci.org"
-  git config --global user.name "Travis CI"
+  git config --global user.email "bot@github.com"
+  git config --global user.name "Github Action"
```

Comment:
shouldn't we use platform independent values?
I would skip email and set `user.name` to `Continuous Integration`

Reply:
Done, it doesn't work now anyway because of not handling env variables that contain branch name etc. in GH env